.github/workflows/next-build-multiarch.yml

```diff
@@ -41,19 +44,12 @@ jobs:
         name: "Set up Docker Buildx ${{ matrix.arch }}"
         uses: docker/setup-buildx-action@v2
       -
-        name: "Docker quay.io Login"
+        name: "Login to quay.io"
         uses: docker/login-action@v2
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-      -
-        name: "Docker docker.io Login"
```

Comment:
removed this because I didn't think we need to push to docker.io and shouldn't need authentication to pull images... but test is now failing? https://github.com/eclipse-che/che-dashboard/actions/runs/6223681358/job/16890210564?pr=926